scripts/fix_package_naming.py

```
@@ -0,0 +1,109 @@
+
+import re
+import requests
+import argparse
+from pathlib import Path
+import sys
+
+#########################################################################################
+# Helper functions                                                                      #
+#########################################################################################
+
+def get_canonical_name(name):
```

Ruff (0.12.2)

Expected 2 blank lines, found 1

Add missing blank line(s)

(E302)